### PR TITLE
fix(discover): Add 1px to table row to account for border

### PR DESCRIPTION
**Pull request by lynnagara**

### CODE REVIEW

File: src/sentry/static/sentry/app/views/organizationDiscover/result/table.jsx [hunk 1]
```diff
@@ -178,7 +178,7 @@ export default class ResultTable extends React.Component {
                 rowCount={data.length + 1}
                 columnCount={colCount}
                 fixedRowCount={1}
-                rowHeight={40}
+                rowHeight={41} // Add 1px for border
```
Comment (billyvg) [inline src/sentry/static/sentry/app/views/organizationDiscover/result/table.jsx]: Does this build? This shouldn't be a valid comment

File: src/sentry/static/sentry/app/views/organizationDiscover/result/table.jsx [hunk 2]
```diff
@@ -202,7 +202,7 @@ export default class ResultTable extends React.Component {
 
 const GridContainer = styled(({visibleRows, ...props}) => <Panel {...props} />)`
   height: ${p =>
-    p.visibleRows * 40 +
+    p.visibleRows * 41 +
```
Comment (billyvg) [inline src/sentry/static/sentry/app/views/organizationDiscover/result/table.jsx]: Can we extract these magic numbers into constants?

File: src/sentry/static/sentry/app/views/organizationDiscover/result/table.jsx (hunk 1 above)
Comment (lynnagara) [reply src/sentry/static/sentry/app/views/organizationDiscover/result/table.jsx]: it builds, and my editor marks it as a valid comment :(

File: src/sentry/static/sentry/app/views/organizationDiscover/result/table.jsx (hunk 2 above)
Comment (lynnagara) [reply src/sentry/static/sentry/app/views/organizationDiscover/result/table.jsx]: done

File: src/sentry/static/sentry/app/views/organizationDiscover/result/table.jsx [hunk 3]
```diff
@@ -10,6 +10,8 @@ import InlineSvg from 'app/components/inlineSvg';
 import Panel from 'app/components/panels/panel';
 import {getDisplayValue, getDisplayText} from './utils';
 
+const TABLE_ROW_HEIGHT = 41; // Includes 1px for border
```
Comment (billyvg) [inline src/sentry/static/sentry/app/views/organizationDiscover/result/table.jsx]: We may want to follow up with a "cell height" = 40, and "table row height" = cell height + border height, it'd be more self documenting

**Review by billyvg — APPROVED** [review]

**Pull request merged**